tests/org.eclipse.n4js.hlc.integrationtests/fixture/IDE-1510_Incomplete_API_Implementation/one.api/src/p/A.n4jsd

```diff
@@ -0,0 +1,40 @@
+/*
```

Comment:
It seems that folders `fixture`, `fixtureApiReplacement`, `probands`, etc. in bundles `org.eclipse.n4js.hlc.tests` and `org.eclipse.n4js.hlc.integrationtests` now contain a lot of redundant test data. Was that a mistake?

We should try to avoid this.

Reply:
In fact, not all are copied but only those needed by the jar/n4js-cli tests are copied. But it is true that duplicate is always evil 😈